airbyte-webapp/src/components/connection/CreateConnectionForm/CreateConnectionForm.tsx

```diff
@@ -117,7 +117,7 @@ const CreateConnectionFormInner: React.FC<CreateConnectionPropsInner> = ({ schem
               <CreateControls
                 isSubmitting={isSubmitting}
                 isValid={isValid && !editingTransformation}
-                errorMessage={getErrorMessage(isValid, dirty)}
+                errorMessage={getErrorMessage(isValid, dirty, errors)}
```

Comment:
Do these errors ever have overlap with the other errors this function uses?

Reply:
There is some overlap with `isValid`, but if you look at the updated implementation of this function it only checks if there is that specific stream error; otherwise it works the same way as before. I just pushed a test to verify that getErrorMessage still works as expected.

Reply:
Half tempted to say we should check errors for isValid, but seems like not worth doing.